Packs/CortexXDR/ReleaseNotes/6_1_66.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Added the **alter_events_from_decider** argument, use this argument to obtain **events_from_decider** context path as a list (for improved for playbook automation) rather than a dictionary (The original API raw response). 
```

Comment:
```suggestion
- Added the ***events_from_decider_format*** argument, use this argument to obtain **events_from_decider** context path as a list (for improved for playbook automation) rather than a dictionary (The original API raw response). 
```